src/pocketmine/Player.php

```diff
@@ -2780,18 +2782,46 @@ public function handlePlayerAction(PlayerActionPacket $packet) : bool{
 				}
 				return true;
 			case PlayerActionPacket::ACTION_START_GLIDE:
+				$ev = new PlayerToggleGlideEvent($this, true);
+				$this->server->getPluginManager()->callEvent($ev);
+				if($ev->isCancelled()){
+					$this->sendData($this);
+				}else{
+					$this->setGliding(true);
```

Comment:
I'm unsure if allowing gliding should be the default behaviour here since it isn't implemented yet. Maybe this should be cancelled by default (until elytra are implemented into the core).